react/src/components/MainLayout/WebUISider.tsx

```diff
@@ -85,6 +87,11 @@ const WebUISider: React.FC<WebUISiderProps> = (props) => {
       icon: <CloudUploadOutlined />,
       key: 'data',
     },
+    supportPendingSessionCount && {
```

Comment:
`supportPendingSessionCount` is not related to user-commited image, right? It's better to add a support key for user-commited image support.

Reply:
Oops, I was confused with another PR. Rename completed!